src/extension/extension.ts

```diff
@@ -112,6 +112,8 @@ export class RunmeExtension {
       ? new GrpcSerializer(context, server, kernel)
       : new WasmSerializer(context, kernel)
     this.serializer = serializer
+    kernel.registerCellTerminalState
```

Comment:
what's this?

Reply:
Fixed in 2c69b84a17dcf327bd3ee5e4e305200fb220cece